### ISSUE TITLE
Archive manager does not respect dark mode

### ISSUE THREAD

**Issue by kdwk** [elementary/files#1758]
## What Happened

![Screenshot from 2021-07-16 21-23-04](https://user-images.githubusercontent.com/39268094/125955074-889db899-40ec-49ed-b6ce-54a8c34d3933.png)

## Expected Behavior

UI elements should be dark to fit in with system dark mode

## Steps to Reproduce

1. Set dark mode
2. Open archive manager

## Platform Information

![Uploading Screenshot from 2021-07-14 10-55-34.png…]()

**Comment by jeremypw** [comment]
The archive manager is not part of the FIles repository - it is actually Gnome FileRoller (https://gitlab.gnome.org/GNOME/file-roller) It is lauched via a contract (https://github.com/elementary/file-roller-contract).  You are right it does not currently honour the Elementary desktop setting, although strangely if launched from the commandline with GTK_DEBUG=interactive, it does honor the "dark variant" setting in the interactive debugger.

**Comment by cassidyjames** [comment]
@jeremypw that's because the elementary OS dark style doesn't force-override the environment, but requires apps to opt in.

@danrabbit is the solution here to package and patch File Roller like we do for Evince?

**Comment by danrabbit** [comment]
Yeah, if File Roller behaves in a Flatpak that's probably the best option. There's C code in the Web patch for listening to dark style so it shouldn't be a difficult patch to write

Gonna move to os-patches
